client/src/pages/SignInModal.tsx

```diff
@@ -43,84 +49,87 @@ const SignInModal: FunctionComponent = () => {
     return <Redirect to={redirect} />;
   }
   return (
-    <CommonModal
-      title={modalTitle}
-      subtitle={subtitle}
-      show={show}
-      handleClose={handleClose}
-      body={
-        <div>
-          <form onSubmit={handleClick}>
-            <div>
-              <div className="row">
-                <div className="text signup">Email Address</div>
-                <div className="text error">{errors.email}</div>
-              </div>
-              <input
-                name="email"
-                placeholder="Enter your company email"
-                type="text"
-                value={email}
-                className={errors.email ? "text-field-input error" : "text-field-input"}
-                onChange={onChangeEmail}
-              />
-            </div>
-            <div>
-              <div className="row">
-                <div className="text signup">
-                  Password
+    <React.Fragment>
```

Comment:
Sorry there's a big diff here! I wrapped everything in `React.Fragment`, so everything was reindented.

My only changes here were:
+ wrap everything in `React.Fragment`
+ show ConfirmationModal if a confirmation email was resent